protocol-units/settlement/mcr/contracts/upgrader/README.md

```diff
@@ -0,0 +1,45 @@
+# Upgrader
```

Comment:
Add a guide here that would essentially allow someone to start from zero and test e2e.